docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -323,7 +323,141 @@ Use this to produce a PDF of the band structure:
            ...: bands = load_node(<PK>)  # PK of the `band_structure` node
            ...: bands.show_mpl(y_origin=fermi_energy, plot_zero_axis=True)
 
+Querying the database
+---------------------
+
+As you will use AiiDA to run your calculations, the database that stores all the data and the provenance will quickly grow to be very large.
+To help you find the needle that you might be looking for in this big haystack, we need an efficient search tool.
+AiiDA provides a tool to do exactly this: the ``QueryBuilder``.
+The ``QueryBuilder`` acts as the gatekeeper to your database, to whom you can ask questions about the contents of your database (also referred to as queries), by specifying what are looking for.
+In the final part of the tutorial, we will show an short demo on how to use the ``QueryBuilder`` to make these queries and understand/use the results.
+
+First, we'll import a set of previously created data entries that we'll use in this section:
+
+.. code-block:: console
+
+    $ verdi import https://object.cscs.ch/v1/AUTH_b1d80408b3d340db9f03d373bbde5c1e/marvel-vms/tutorials/aiida_tutorial_2020_07_perovskites_v0.9.aiida
+
+To help you organise your data, AiiDA allows you to *group* nodes together.
+Let's have a look at the groups we've imported from the archive above:
+
+.. code-block:: console
+
+    $ verdi group list
+    Info: to show groups of all types, use the `-a/--all` option.
+      PK  Label            Type string    User
+    ----  ---------------  -------------  ---------------
+       6  tutorial_pbesol  core           aiida@localhost
+       7  tutorial_lda     core           aiida@localhost
+       8  tutorial_pbe     core           aiida@localhost
```

Comment:
Maybe I would mention above or below what this groups contain? From the queries before I deduce these contain at least the PwCalcjobs, but only these? All the nodes involved in the workflow? We could maybe add a -C to illustrate the size of a tipical database.

Reply:
Adding `-C` and a quick phrase to elucidate their content is a good idea 👌 .